.circleci/config.yml

```diff
@@ -115,7 +118,10 @@ jobs:
           command: |
             export AWS_ACCESS_KEY_ID=${PROD_AWS_ACCESS_KEY_ID}
             export AWS_SECRET_ACCESS_KEY=${PROD_AWS_SECRET_ACCESS_KEY}
-            aws s3 sync --delete --exclude "*le-components/*" --acl public-read packages/docs/next-doc-site/build s3://${BUCKET}.new.datacamp.com/
+            aws s3 rm s3://${BUCKET}.new.datacamp.com/ --exclude "*le-components/*" --recursive
+            aws s3 sync packages/docs/next-doc-site/build s3://${BUCKET}.new.datacamp.com/ --acl public-read --exclude "*.js" --exclude "*.svg" --exclude "*.png" --exclude "*.jpg" --exclude "*.woff" --exclude "*.woff2" --cache-control public,max-age=0,must-revalidate
+            aws s3 sync packages/docs/next-doc-site/build s3://${BUCKET}.new.datacamp.com/ --acl public-read --exclude "*" --include "*.js" --cache-control public,max-age=31536000,immutable
+            aws s3 sync packages/docs/next-doc-site/build s3://${BUCKET}.new.datacamp.com/ --acl public-read --exclude "*" --include "*.svg" --include "*.png" --include "*.jpg" --include "*.woff" --include "*.woff2" --cache-control public,max-age=31536000,immutable,no-transform
```

Comment:
Can we version all the resource URLs that get a long max-age (either through directory structure or a hash in the file name) so we can easily update them?